chcemvediet/apps/anonymization/cron.py

```diff
@@ -5,6 +5,7 @@
 
 import subprocess32
```

Comment:
Komentar sa tyka binarneho `loremipsum.pdf.`
Kde mozem ulozit tento subor?

Reply:
Do `chcemvediet/apps/anonymization/mocks/normalized.pdf`. Urcite nie do `static` ako mas teraz, pretoze subory v adresare static su verejne downloadnutelne z webu.

Reply:
Thread oznacujem ako resolved, kedze to spravime v samostatnom PR.